src/main/kotlin/org/rust/cargo/project/model/impl/CargoProjectImpl.kt

```diff
@@ -518,6 +520,18 @@ data class CargoProjectImpl(
             possiblePackages.any { workspace.findPackageByName(it) != null }
     }
 
+    fun findStdlibInBazelWorkspace(): File? {
+        val stdlibPath = stdlibPathBazel()
+        return if (stdlibPath != null && Files.exists(stdlibPath)) stdlibPath.toFile() else null
+    }
+
+    fun stdlibPathBazel(): Path? {
+        val workspaceRoot: CargoWorkspace.Package = rawWorkspace?.findRootPackage() ?: return null
+        val projectName = workspaceRoot.rootDirectory.fileName.toString()
+        // TODO: fragile + OS dependent
+        return Path.of(workspaceRoot.rootDirectory.toString(), "bazel-$projectName/external/rust_darwin_x86_64/lib/rustlib/src/library")
```

Comment:
Each Bazel project has build tools, package managers etc. declared in a `WORKSPACE` file. For a Rust project it'll include the Rust stdlib, which is downloaded to a path accessible from `{projectRoot}/bazel-$projectName`.

The current code is fragile - we should be communicating the precise path of the stdlib from the Bazel plugin somehow rather than relying on knowing it here.